.github/ISSUE_TEMPLATE/java-stable-release-template.md

```diff
@@ -0,0 +1,43 @@
+### RELEASE_VERSION_NUMBER-alpha.1 pre-release checklist 
+
+- [ ] Update the SNAPSHOT version in this repo's `README.md` file 
+- [ ] Change version name to `RELEASE_VERSION_NUMBER-alpha.1` in this repo's `gradle.properties` (make sure to remove `-SNAPSHOT`)
+- [ ] Create a new release in Github with a release title of `Mapbox Java SDK vRELEASE_VERSION_NUMBER-alpha.1` and a tag version of `vRELEASE_VERSION_NUMBER-alpha.1`. This will publish artifacts to Bintray.
+- [ ] Change version name back to having `-SNAPSHOT` at the end
+- [ ] Update this repo's `CHANGELOG.md`
+
+### RELEASE_VERSION_NUMBER-alpha.1 pre-release testing
+- [ ] Create a pull request in [`/mapbox-gl-native`](https://github.com/mapbox/mapbox-gl-native) that updates the Mapbox Maps SDK for Android's Java SDK dependency to `RELEASE_VERSION_NUMBER-alpha.1`
+- [ ] Create a pull request in [the Mapbox Navigation SDK for Android repo](https://github.com/mapbox/mapbox-navigation-android) that updates the Nav SDK's Java SDK dependency to `RELEASE_VERSION_NUMBER-alpha.1`
+- [ ] Report any regressions on this ticket
+
+### RELEASE_VERSION_NUMBER-alpha.2 pre-release checklist 
```

Comment:
Would we like to perform 2 pre-releases? I think changes in this repository flow in a rather slow pace where one pre-release should suffice. Any thoughts @mapbox/maps-android?

Reply:
Yea, I don't think we need to do 2 pre-releases.

Reply:
Have removed the `alpha.2` section.